paybutton/src/index.tsx

```diff
@@ -104,7 +104,7 @@ function findAndRender <T>( className: string, Component: React.ComponentType<an
 
       if ( ! requiredProps.every( name => name in attributes ) ) {
         console.error( 'PayButton: missing required attribute: ' + JSON.stringify( requiredProps.filter( name => ! ( name in attributes ) ) ) );
-        return;
+        // return;
```

Comment:
Meant to be commented out still?

Reply:
yes,  for someone in the future to be able to see that and easily revert if needed. versus having to dig through git history.